test/scheduler/assessment/NotifyAssessmentCentrePassedOrFailedJobSpec.scala

```diff
@@ -32,7 +32,8 @@ class NotifyAssessmentCentrePassedOrFailedJobSpec extends PlaySpec with MockitoS
 
   "process next assessment centre passed or failed application" should {
     "work" in new TestFixture {
-      Job.tryExecute().futureValue mustBe ()
+      val unit = ()
```

Comment:
Since this is the second use in this PR, and we'll likely use it in a lot more places, can we define it somewhere central?